src/type/introspection.d.ts

```diff
@@ -35,6 +35,6 @@ export const SchemaMetaFieldDef: GraphQLField<any, any>;
 export const TypeMetaFieldDef: GraphQLField<any, any>;
 export const TypeNameMetaFieldDef: GraphQLField<any, any>;
 
-export const introspectionTypes: ReadonlyArray<GraphQLType>;
+export const introspectionTypes: ReadonlyArray<GraphQLNamedType>;
 
 export function isIntrospectionType(type: GraphQLType): boolean;
```

Comment:
Can you change this to take GraphQLNamedType too (which is how the actual function is declared in Flow)?

Reply:
Is it, but looks to me like it'd work with any `GraphQLType` and making the API more strict would break backward compatibility?

Reply:
Hmm? The implementation lolks